torch/ao/nn/intrinsic/modules/fused.py

```diff
@@ -144,3 +145,13 @@ def __init__(self, linear, tanh):
             'Incorrect types for input modules{}{}'.format(
                 type(linear), type(tanh))
         super().__init__(linear, tanh)
+
+class ConvAdd2d(_FusedModule):
+    r"""This is a sequential container which calls the Conv2d modules with extra Add.
+    During quantization this will be replaced with the corresponding fused module."""
+    def __init__(self, add, conv):
```

Comment:
nit: can we swap the order of conv and add, to be more consistent with the other ops

Reply:
Good suggestion, changed the order to make it more consistent.